torchvision/prototype/features/_bounding_box.py

```diff
@@ -40,6 +40,9 @@ def __new__(
     def to_format(self, format: Union[str, BoundingBoxFormat]) -> "BoundingBox":
         # import at runtime to avoid cyclic imports
         from torchvision.prototype.transforms.functional import convert_bounding_box_format
+        # I think we can avoid this by not having a `to_format` method but instead require users to explicitly call the
```

Comment:
This is for convenience and I expect the most usages in application code that we don't provide. This is similar to `Tensor.to()` to change the device or dtype. It would be rather inconvenient to do something like `torch.to(Tensor())` all the time.

Still, I agree with the avoidance of lazy imports here since they point to a design problem. This was also observed in https://github.com/pytorch/vision/pull/5295#discussion_r793619498. My solution would be to move the conversion implementation here and only expose it through this method as well through a transform.

Reply:
Let's omit for now any convenience methods if they cause such issues and we can revise once the API is complete.